easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.2.4-intel-2015a.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Modified by: Robert Schmidt <rjeschmi@gmail.com>
+# Ottawa Hospital Research Institute - Bioinformatics Team
```

Comment:
here is an example where my type of header is more readable, this situation has been considered.
You know where to find it ;-)
btw. I'd like at least the original author, Cedric, to be also in the list just as a means of thanks...